src/components/graph-view-props.js

```diff
@@ -74,7 +74,8 @@ export type IGraphViewProps = {
   onSwapEdge: (sourceNode: INode, targetNode: INode, edge: IEdge) => void,
   onUndo?: () => void,
   onUpdateNode: (node: INode) => void,
-  onZoomEnd?: () => void,
+  onZoomCall: (viewTransform: object) => void,
```

Comment:
onZoomStart to be symmetrical?

Reply:
Yeah I wasn't sure if that was misleading because there's a `handleZoomStart` method, which we wouldn't actually be calling it in. It does still initiate the zoom call, though, so it probably is fine, good call.